src/InventiveContent.tsx

```diff
@@ -16,13 +16,14 @@ export const InventiveContent = (props: InventiveContentProps) => {
     if (!urlInfo) return;
 
     const targetOrigin = new URL(urlInfo.url).origin;
+    const reference = window.location.href;
 
     const iframe = iframeRef.current;
     const postMessageToIframe = () => {
       if (iframe?.contentWindow) {
         iframe.contentWindow.postMessage(
-          createEmbedTokensMessage(urlInfo.tokens),
-          targetOrigin,
+          createEmbedTokensMessage(urlInfo.tokens, reference),
```

Comment:
`reference` is to generic a name for my test. Let's go for something that's more to the point. Don't have good suggestions at the moment other than names like: `embeddingUrl`, `parentUrl` - both admittedly are kinda lame.

Reply:
Because the function name is `createEmbedTokenMessage`, I didn't want to give it embed/parent context.I think the `reference` would give a sense of where it is generated from. How about `referenceUrl`?